docs/raw/push-notification-server.md

```diff
@@ -126,13 +124,14 @@ message PushNotificationDeviceToken {
   TokenType token_type = 1;
   string token = 2;
   string installation_id = 3;
-  PushNotificationPreferences preferences = 4;
+  PushNotificationFilterSettings filter_settings = 4;
 }
 
 message PushNotificationPreferences {
-  repeated DeviceToken device_tokens = 1;
+  repeated PushNotificationOptions options = 1;
   uint version = 2;
   boolean unregister = 3;
+  string access_token = 4;
```

Comment:
`access_token` should perhaps not be bundled under `PushNotificationPreferences`. Consider renaming or splitting message.

Reply:
I might do that in a separate PR if it's ok, I will be getting rid of a few messages in order to simplify and we can shuffle things around at that point if that's ok.